app/src/main/java/com/eventyay/organizer/core/organizer/detail/OrganizerDetailView.java

```diff
@@ -6,4 +6,12 @@
 import com.eventyay.organizer.common.mvp.view.Refreshable;
 import com.eventyay.organizer.data.user.User;
 
-public interface OrganizerDetailView extends Progressive, Erroneous, Refreshable, ItemResult<User> { }
+public interface OrganizerDetailView extends Progressive, Erroneous, Refreshable, ItemResult<User> {
+    @Override
+    void showProgress(boolean show);
+
+    @Override
+    void showError(String error);
```

Comment:
Both of the above methods arent needed. They are already there

Reply:
Removed